src/app/reactUtils.ts

```diff
@@ -249,3 +266,41 @@ export const disableContentTabbingHandler = (isEnabled: boolean) => () => {
 export const useDisableContentTabbing = (isEnabled: boolean) => {
   React.useEffect(disableContentTabbingHandler(isEnabled), [isEnabled]);
 };
+
+export type KeyCombinationOptions = Partial<Pick<KeyboardEvent, 'altKey' | 'ctrlKey' | 'key' | 'metaKey' | 'shiftKey'>>;
+
+/**
+ * Attach keydown event listener to the document and check if clicked keys are matching @param options
+ * Value of @param options.key is converted to lowercase before comparing.
+ * @param {KeyCombinationOptions} options
+ * @param {Function} callback
+ */
+export const useKeyCombination = (
+  options: KeyCombinationOptions,
+  callback: () => void,
+  noopHandler?: (activeElement: Element | null) => boolean
+) => {
+  const document = assertDocument();
```

Comment:
you're doing a lot of document stuff, i feel like you don't need that.

on focusin you can use event.target instead of document.activeElement

you could bind TWO handlers, one on the content and one of the cardwrapper, that each do ONE thing instead of switching based on complex logic

Reply:
> you could bind TWO handlers, one on the content and one of the cardwrapper, that each do ONE thing instead of switching based on complex logic

So one `useKeyCombination` on `<PageContent>` and the second on `<CardWrapper>`?
I'm not convinced to use it - It would split the focusing logic into one more place and require using redux to trigger focus in the card when we can have everything in one place
Splitting code into smaller parts is a good thing, but it will require us to remember of all places which can be affected

If we add a method like `highlighter.focus()` and pass `event` to `moveFocuse` then the "complex logic" from `CardWrapper` could become:
```ts
const moveFocus = React.useCallback((event: KeyboardEvent) => {
    if (!focusedHighlight || !event.target || !element.current) {
      return;
    }

    if (element.current.contains(event.target)) {
        highlighter.focus(focusedHighlight);
        setShouldFocusCard(false); // I think it can be also removed since we have `onFocusLost` in this component that does the same
    } else {
      setShouldFocusCard(true);
    }
  }, [element, focusedHighlight]);